src/main/java/org/cryptomator/cryptofs/health/dirid/OrphanDir.java

```diff
@@ -12,6 +32,20 @@
  */
 public class OrphanDir implements DiagnosticResult {
 
+	private static final String FILE_PREFIX = "file";
+	private static final String DIR_PREFIX = "directory";
+	private static final String SYMLINK_PREFIX = "symlink";
+	private static final String LONG_NAME_SUFFIX_BASE = "_withVeryLongName";
+	private static final MessageDigest SHA1_HASHER;
```

Comment:
This class is not threadsafe. Create a new one within `fix(...)`.

Reply:
Fixed in 833d613